config/application.php

```diff
@@ -55,6 +55,7 @@
 	'RequireEmailConfirm'		=> false,					// Require e-mail confirmation during registration.
 	'RequireChangeConfirm'		=> false,					// Require confirmation when changing e-mail addresses.
 	'EmailConfirmExpire'		=> 48,						// E-mail confirmations expire hours. Unconfirmed accounts will expire after this period of time.
+	'AntiRegesterationSpam'		=> false,					// Work only when 'RequireEmailConfirm' is active , if the ip have an unconfirmed account , it would privent the user from registering new account.
```

Comment:
'AntiRegesterationSpam' -> 'PendingRegistration'
// Requires 'RequireEmailConfirm' to be true. Prevents new registration if ip address has a pending registration.